### PR TITLE
Remove pluralization on profile#show

### PR DESCRIPTION
**Pull request by farukaydin**
Fixes https://github.com/rubygems/rubygems.org/issues/1018

I'm not sure it is necessary to add test for it, but I added anyway.

cc: @qrush

### CODE REVIEW

**Comment by qrush** [comment]
It's not dynamic. :+1: to remove the test, we don't need to test static copy like that.

**Comment by farukaydin** [comment]
@qrush test is removed. :tada: :roller_coaster:

**Comment by farukaydin** [comment]
Encounter with rubocop style guide error. I'll modified it soon.

File: test/unit/helpers/pages_helper_test.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class PagesHelperTest < ActionView::TestCase
     setup do
       @rubygem = create(:rubygem, name: "rubygems-update")
       @version_first = create(:version, number: "1.4.8", rubygem: @rubygem)
-      @version_last = create(:version, number: "2.4.8", built_at: DateTime.new(2015,1,1), rubygem: @rubygem)
+      @version_last = create(:version, number: "2.4.8", built_at: DateTime.new(2015, 1, 1), rubygem: @rubygem)
```
Comment (sferik) [inline test/unit/helpers/pages_helper_test.rb]: I just fixed this in `master` too. You should be able to rebase to pick up that change.

Comment (farukaydin) [inline test/unit/helpers/pages_helper_test.rb]: @sferik thanks. Rebased and pushed again.

**Pull request merged**